### PR TITLE
Use approach that is consistent with the rest of flags mgmt for handler.

### PR DESCRIPTION
**Pull request by ericbottard**

### CODE REVIEW

File: riff-cli/cmd/init.go [hunk 1]
```diff
@@ -160,18 +159,13 @@ func InitGo(initOptions *options.InitOptions) (*cobra.Command, *options.InitOpti
...
-			err := initializers.Go().Initialize(*initOptions)
-			if err != nil {
-				return err
-			}
-			return nil
+			return initializers.Go().Initialize(*initOptions)
 		},
 	}
-	initGoCmd.Flags().String("handler", "", "the name of the function handler")
+	initGoCmd.Flags().StringVar(&initOptions.Handler, "handler", "", "the name of the function handler (Exported go symbol)")
```
Comment (trisberg) [inline riff-cli/cmd/init.go]: What does "Exported go symbol" mean here?

Comment (ericbottard) [reply riff-cli/cmd/init.go]: That the handler should be the name of the Exported go symbol in the go plugin

Comment (ericbottard) [reply riff-cli/cmd/init.go]: Maybe change to "Exported go function name"?

Comment (ericbottard) [reply riff-cli/cmd/init.go]: or "name of Exported go function"?

Comment (trisberg) [reply riff-cli/cmd/init.go]: yeah, that sounds better - can change on merge

**Comment by trisberg** [comment]
Rebased and merged as commit 210fe186a1fc518cb24024b448c047cc1dd09c1a

**Pull request closed without merging**